Modules/Customer/Database/Migrations/2023_03_04_141823_create_customers_table.php

```diff
@@ -15,10 +15,21 @@ public function up()
         Schema::create('customers', function (Blueprint $table) {
             $table->id();
             $table->unsignedBigInteger('user_id')->nullable();
+            $table->unsignedBigInteger('retailer_id')->nullable();
```

Comment:
@arijit1111 Kindly create new migration with update table action. Then add all of the required changes in the existing database table over there. Kindly revert all of these changes.